main.py

```diff
@@ -0,0 +1,183 @@
+from fastapi import (
+    FastAPI,
+    Depends,
+    HTTPException,
+    status,
+    security
+)
+from sqlalchemy.orm import Session
+
+import crud, schemas, utils
+
+
+app = FastAPI()
+
+
+@app.get("/")
+def root():
+    return {"Hello": "World"}
```

Comment:
why do u need this ?

Reply:
oops, used it when did alot of quicktests. will be removed